source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRImageRenderer.mm

```diff
@@ -48,22 +49,75 @@ - (CGSize) getImageSize:(std::shared_ptr<Image> const &) imgElem
+                                     attribute:NSLayoutAttributeCenterY
+                                    multiplier:1
+                                      constant:0]];
+
+    switch (alignment)
+    {
         case HorizontalAlignment::Center:
-            return UIViewContentModeCenter;
+        {
+            [constraints addObject: 
+                [NSLayoutConstraint constraintWithItem:superview
+                                             attribute:NSLayoutAttributeCenterX
+                                             relatedBy:NSLayoutRelationEqual
+                                                toItem:view
+                                             attribute:NSLayoutAttributeCenterX
+                                            multiplier:1
+                                              constant:0]];
+                return constraints;
+        }
         case HorizontalAlignment::Left:
-            return UIViewContentModeLeft;
+        {
+            [constraints addObject: 
+                [NSLayoutConstraint constraintWithItem:superview
+                                             attribute:NSLayoutAttributeLeading
+                                             relatedBy:NSLayoutRelationEqual
+                                                toItem:view
+                                             attribute:NSLayoutAttributeLeading
+                                            multiplier:1
+                                              constant:0]];
+            return constraints;
+        }
         case HorizontalAlignment::Right:
-            return UIViewContentModeRight;
+        {
+            [constraints addObject: 
+                [NSLayoutConstraint constraintWithItem:superview
+                                             attribute:NSLayoutAttributeTrailing
+                                             relatedBy:NSLayoutRelationEqual
+                                                toItem:view
+                                             attribute:NSLayoutAttributeTrailing
+                                            multiplier:1
+                                              constant:0]];
+            return constraints;
+        }
         default:
-            return UIViewContentModeLeft;
+        {
+            [constraints addObject: 
+                [NSLayoutConstraint constraintWithItem:superview
+                                             attribute:NSLayoutAttributeLeading
+                                             relatedBy:NSLayoutRelationEqual
+                                                toItem:view
+                                             attribute:NSLayoutAttributeLeading
+                                            multiplier:1
+                                              constant:0]];
+            return constraints;
+        }
     }
+    return constraints;
 }
-- (UIView* ) render: (UIStackView*) viewGroup
+-(UIView*)render: (UIView*) viewGroup
```

Comment:
[nit] Now this one is not consistent with the rest of the file.
Does XCode have some auto formatting functionality you could leverage to avoid having to look at these by hand?